features/bootstrap/FeatureContext.php

```diff
@@ -95,14 +132,51 @@ public function stdoutShouldReturnExactly($output_stream, PyStringNode $expected
 	 */
 	public function stdoutShouldReturnSomethingLike($output_stream, PyStringNode $expected_output)
 	{
-		$command_output = $output_stream === "STDOUT" ? $this->command->stdout : $this->command->stderr;
+		$command_output = $output_stream === "STDOUT" ? $this->commands[0]->stdout : $this->commands[0]->stderr;
 
 		$expected_out = isset($expected_output->getStrings()[0]) ? $expected_output->getStrings()[0] : '';
 		if (strpos($command_output, $expected_out) === false) {
 			throw new Exception("Actual output is:\n" . $command_output);
 		}
 	}
 
+
+	/**
+	 * @Then /(STDOUT|STDERR) of command :index should return something like/
+	 */
+	public function stdoutOfCommandShouldReturnSomethingLike($output_stream, $index, PyStringNode $expected_output)
+	{
+		$command_output = $output_stream === "STDOUT" ? $this->commands[ $index - 1 ]->stdout : $this->commands[ $index - 1 ]->stderr;
+
+		$expected_out = isset($expected_output->getStrings()[0]) ? $expected_output->getStrings()[0] : '';
+		if (strpos($command_output, $expected_out) === false) {
+			throw new Exception("Actual output is:\n" . $command_output);
+		}
+	}
+
+	/**
+	 * @Then The site :site should be :type multisite
+	 */
+	public function theSiteShouldBeMultisite( $site, $type )
+	{
+		$php_container = implode( explode( '.', $site ) ) . '_php_1';
```

Comment:
Instead of evaluating the name of the container and doing `docker exec`, changing directory to the site root and then doing `docker-compose exec` could be better and less error-prone.

Reply:
Yup. Good idea!